src/components/PlanetPage.tsx

```diff
@@ -2,33 +2,37 @@ import { useState } from "react";
 import SubMenu from "./SubMenu";
 import SubPage from "./SubPage";
 
-export type PlanetType = {
```

Comment:
The `Type` suffix can be omitted. It does not add much. (Same with the `I` prefix for interfaces.)